source/help/getting-started/setting-your-status-availability.rst

```diff
@@ -0,0 +1,58 @@
+Setting Your Status and Availability
+====================================
+
+Let your team know whether you're available by setting a custom status and your availability in Mattermost.
+
+Setting a Custom Status
+-----------------------
+
+Set a custom status to add an optional emoji to a descriptive status message. Other members can see your status anywhere they can see your name, such as the left-hand sidebar, within conversations, and within Direct Messages. 
+
+1. To set a custom status, select your avatar at the top of the left-hand sidebar, then select **Set a Custom Status**.
+
+.. image:: ../../images/set-custom-status.png
+  :alt: Set a custom status.
+
+2. Choose from a list of suggested statuses, reuse a recent status, or enter a new emoji and status, then select **Set Status**. The Speech bubble emoji 💬  is used by default if you don't specify an emoji. A custom status can be a maximum of 100 characters in length.
+
+.. image:: ../../images/Custom-Status-Marketing-Animation.gif
+  :alt: Set a custom status that includes an optional emoji and a descriptive message.
+
+.. note::
+
+  - If custom statuses aren't available, you can request your System Admin to enable this feature in **System Console > Site Configuration > Users and Teams > Enable Custom Statuses**.
+  - Custom statuses will be available in the Mattermost Mobile App in a future release. 
+
+Clearing a Custom Status
+------------------------
+
+To clear a custom status, select your avatar at the top of the left-hand sidebar, then select **Clear Status**, or select the **Clear** option next to your current status.
+
+.. image:: ../../images/clear-custom-status.png
+  :alt: Clear your custom status.
+
+Setting Your Availability
+-------------------------
+
+To set your availability, select your avatar, then specify your availability as  **Online**, **Away**, **Do Not Disturb**, or **Offline**.
+
+.. image:: ../../images/set-your-availability.png
+  :alt: Set your availability to online, away, do not disturb, or offline.
+
+When you set your availability, an availability icon displays next to your name. Setting your availability as **Do Not Disturb** automatically disables desktop, email, and push notifications.
+
+Other members can see your availability anywhere they can see your name, such as the left-hand sidebar, within conversations, and within Direct Messages. 
```

Comment:
left-hand > channel

Reply:
Fixed. Docs ticket https://mattermost.atlassian.net/browse/MM-33689 logged to review and update the full doc set.